src/twisted/web/http.py

```diff
@@ -2048,22 +2071,30 @@ def allContentReceived(self):
 
         self._handlingRequest = True
 
-        # Pause the producer if we can. If we can't, that's ok, we'll buffer.
-        if not self._waitingForTransport:
-            self._networkProducer.pauseProducing()
-
         req = self.requests[-1]
         req.requestReceived(command, path, version)
 
 
-    def rawDataReceived(self, data):
-        self.resetTimeout()
-
+    def dataReceived(self, data):
+        """
+        Data was received from the network.  Process it.
+        """
         # If we're currently handling a request, buffer this data. We shouldn't
```

Comment:
This comment seems to be seriously misleading now, could you update it to describe the new behaviour? (We have _not_ necessarily paused the transport yet)